docs/src/stories/components/Layout/LayoutBeta.stories.jsx

```diff
@@ -0,0 +1,485 @@
+      table: {
+        category: 'Responsive'
+      }
+    },
+
+    // Pane
+
+    paneWidth: {
+      options: ['default', 'narrow', 'wide'],
+      control: {
+      type: 'inline-radio'
+      },
+      description: 'Defines the width of the pane',
+      table: {
+        category: 'Pane'
+      }
+    },
+    panePosition: {
+      options: ['start', 'end'],
+      control: {
+        type: 'inline-radio'
+      },
+      description: 'Defines the position of the pane. `start` renders the pane on the left, and `end` renders it on the right.',
+      table: {
+        category: 'Pane',
+      }
+    },
+    paneResponsivePosition: {
+      options: ['inherit', 'start', 'end'],
+      control: {
+        type: 'inline-radio',
+      },
+      description: 'If `responsiveVariant` is set to `stackRegions`, defines the position of the pane in the responsive variant. `start` puts the pane above `content`, and `end` puts it below `content`. `inherit` uses the same value from `panePosition`.',
+      table: {
+        category: 'Pane'
+      }
+    },
+    paneDivider: {
+      control: { type: 'boolean' },
+      description: 'Whether to show a pane line divider.',
+      table: {
+        category: 'Pane'
+      }
+    },
+    paneResponsiveDivider: {
+      options: ['none', 'line', 'filled'],
+      control: {
+        type: 'inline-radio'
+      },
+      description: 'Whether to show a divider between `pane` and `content` regions if `responsiveVariant` is set to `stackRegions`. If `pane` appears above `content`, a `(...)-divider` class will be placed in the `pane` region. Otherwise it will be placed in the `content` region.',
+      table: {
+        category: 'Pane'
+      }
+    },
+    paneIsSticky: {
+      control: { type: 'boolean' },
+      description: 'Whether to make the pane sticky. Don’t use it in the presence of footer regions.',
+      table: {
+        category: 'Pane'
+      }
+    },
+
+    // Content
+
+    contentWidth: {
+      options: ['fluid', 'sm', 'md', 'lg', 'xl'],
+      control: {
+        type: 'inline-radio'
+      },
+      description: 'Defines the maximum width of the content region. `fluid` sets it to full-width. Other values follow container widths from `sm` to `xl`. With smaller widths, the content region will try to stay centered to the viewport area.',
+      table: {
+        category: 'Content'
+      }
+    },
+
+
+    // Header
+
+    hasHeader: {
+      control: { type: 'boolean' },
+      table: {
+        category: 'Header'
+      }
+    },
+
+    headerDivider: {
+      control: { type: 'boolean' },
+      description: 'Whether to show a header divider',
+      table: {
+        category: 'Header'
+      }
+    },
+
+    headerResponsiveDivider: {
+      options: ['none', 'line', 'filled'],
+      control: {
+        type: 'inline-radio'
+      },
+      description: 'Whether to show a divider below the `header` region if in responsive mode.',
```

Comment:
To me, "responsive mode" refers to when we're at a breakpoint where the "responsiveVariant" is active. Is this correct?

Reply:
@mperrotti this was some old copy I forgot to review. Fixed it!